docs/rules/forbid-component-props.md

```diff
@@ -35,10 +37,94 @@ The following patterns are not considered warnings:
 
 ```js
 ...
-"forbid-component-props": [<enabled>, { "forbid": [<string>] }]
+"forbid-component-props": [<enabled>, { "forbid": [<string>|<object>], "defaultAllowOnComponents": <bool>, "defaultAllowOnDOM": <bool>}]
```

Comment:
unless you're suggesting that they'd be top-level settings as well? if so, i do think i like that.